app/hotels/src/singleHotel/SingleHotelContainer.js

```diff
@@ -103,21 +93,25 @@ export default class SingleHotelContainerWithContext extends React.Component<
       checkout,
       roomsConfiguration,
       hotelId,
-      currency,
+      currency: this.props.currency,
```

Comment:
This is maybe not necessary if this lives in `this.props` since you are using `...this.props` below.

Reply:
done